packages/blade/rollup-themes.config.js

```diff
@@ -0,0 +1,40 @@
+import { babel as pluginBabel } from '@rollup/plugin-babel';
+import pluginPeerDepsExternal from 'rollup-plugin-peer-deps-external';
+import pluginResolve from '@rollup/plugin-node-resolve';
+import pluginCommonjs from '@rollup/plugin-commonjs';
+
+const webExtensions = [
+  '.web.js',
+  '.web.ts',
+  '.web.tsx',
+  '.desktop.js',
+  '.desktop.ts',
+  '.desktop.tsx',
+  '.mobile.js',
+  '.mobile.ts',
+  '.mobile.tsx',
+  '.js',
+  '.ts',
+  '.jsx',
+  '.tsx',
+  '.mjs',
+];
+
+export default {
```

Comment:
why do you need this? you can add this config to the existing rollup config too?

Reply:
Makes sense. Done